scripts/common/sonic-telemetry-build/build.sh

```diff
@@ -12,6 +12,12 @@ sudo sed -ri 's/^unixsocketperm .../unixsocketperm 777/' /etc/redis/redis.conf
 sudo sed -ri 's/redis-server.sock/redis.sock/' /etc/redis/redis.conf
 sudo service redis-server start
 
+# Install libyang
+sudo dpkg -i buildimage/target/debs/stretch/libyang*.deb
+
+# Clone sonic-mgmt-framework repository
+git clone https://github.com/Azure/sonic-mgmt-framework
```

Comment:
How is it used?

Reply:
libyang is for validation of the configuration as described in https://github.com/Azure/SONiC/blob/master/doc/mgmt/Management%20Framework.md#3228-config-validation-library-cvl

Reply:
I see `sonic-mgmt-framework` repo not used

Reply:
Code inside mgmt-framework is shared with telemetry repo. telemetry repo will not compile if the mgmt-framework is not there.

Reply:
`sonic-telemetry` repo is not self-contained, and this dependency is not mentioned in [doc](https://github.com/Azure/sonic-telemetry/#installing). Please consider improve the source code management, for example using git submodule, etc.

I will approve it to unblock.